scripts/create_test

```diff
@@ -257,7 +257,7 @@ OR
         args.no_batch = True
```

Comment:
Around line 180, please update the help field for --test-id to explain the added random component.